### PR TITLE
added css to block pointer events

### PR DESCRIPTION
**Pull request by bkd705**
POC for blocking out events and "freezing page" by using CSS to disable pointer events

## Summary by CodeRabbit

- **New Features**
	- Enhanced editor mode functionality with improved setup and teardown processes.
	- Introduced style management to prevent user interactions during editing.

- **Bug Fixes**
	- Improved cleanup process when exiting editor mode.

### CODE REVIEW

File: apps/mocksi-lite/content/EditMode/editMode.ts [hunk 1]
```diff
@@ -9,16 +9,27 @@ import { decorate } from "./decorator";
...
-		blockClickableElements();
-		document.body.addEventListener("dblclick", onDoubleClickText);
-		return;
+const setupEditor = async (recordingId?: string) => {
+	sendMessage("attachDebugger");
+	if (recordingId) {
+		await chrome.storage.local.set({ [MOCKSI_RECORDING_ID]: recordingId });
 	}
 
+	// blockClickableElements();
```
Comment (elg0nz) [inline apps/mocksi-lite/content/EditMode/editMode.ts:24]: nit: was commenting this out intentional?

**Review by elg0nz — APPROVED** [review]
LGTM 👍

File: apps/mocksi-lite/content/EditMode/editMode.ts (hunk 1 above)
Comment (bkd705) [reply apps/mocksi-lite/content/EditMode/editMode.ts]: yeah removed it now!

Comment (bkd705) [reply apps/mocksi-lite/content/EditMode/editMode.ts]: we block using the css now this function isnt necessary, ill remove the actual function in a future PR

**Pull request merged**